src/Utilities/GeneralUtils.cpp

```diff
@@ -118,3 +119,18 @@ bool GeneralUtils::HasHealthRatioThresholdChanged(double oldRatio, double newRat
 
 	return false;
 }
+
+bool GeneralUtils::ApplyTheaterSuffixToString(char* str)
+{
+	if (auto pSuffix = strstr(str, "~~~"))
+	{
+		auto theater = ScenarioClass::Instance->Theater;
+		auto pExtension = Theater::GetTheater(theater).Extension;
+		pSuffix[0] = pExtension[0];
+		pSuffix[1] = pExtension[1];
+		pSuffix[2] = pExtension[2];
```

Comment:
```suggestion

		pSuffix[0] = pExtension[0];
		pSuffix[1] = pExtension[1];
		pSuffix[2] = pExtension[2];

```
Also can't this be done via a function call? Also IMO should emit a warning if the amount of `~` symbols doesn't equate to 3, which is a developer error.

Reply:
I'm not sure if this is really needed here